Ecosystem Partners/Marketplace Guides/getting-started-with-kemp-technology-loadmaster-template.md

```diff
@@ -0,0 +1,78 @@
+{{{
+  "title": "Getting Started With KEMP Technologies Virtual Loadmaster",
+  "date": "01-19-2017",
+  "author": "Sashi Begur, KEMP Technologies",
+  "attachments": [],
+  "contentIsHTML": false,
+  "sticky": true
+}}}
+
+
+# **Technology Profile**
+KEMP Technologies is the leading provider of application-centric load-balancing solutions, used by thousands of businesses that consider cloud, e-commerce and web applications as mission-critical to their long-term success. KEMP helps companies rapidly grow their business by providing 24/7 application availability, better cloud application performance and secure operations – while streamlining IT costs and maximizing affordability. [KEMP Technologies Website](http://www.kemptechnologies.com)
+
+### **Contact KEMP**
+| Sales Contact | Support Contact |
+|---------------|---------------- |
+| sales@kemptechnologies.com<br>|support@kemptechnologies.com<br>|

```

Comment:
change to `|(mailto:sales@kemptechnologies.com)|(mailto:support@kemptechnologies.com)|`